include/aspect/particle/output/interface.h

```diff
@@ -86,26 +93,52 @@ namespace aspect
                                const double current_time) = 0;
 
           /**
+           * Returns the particle file name taking into account the file index as well as the mpi proccess.
+           */
+          virtual
+          std::string
+          get_file_name () const;
+
+          /**
+           * Get the absolute path where the specified particle output will be generated.
+           */
+          virtual
+          std::string
+          get_particle_output_location () const;
+
+          /**
+           * Get the current file index.
```

Comment:
Please add to the documentation that this function already pads the number with zeroes and returns the index as string. Otherwise somebody might want to use it to get an unsigned int.